proposals/wac-acp-diff-story.md

```diff
@@ -133,3 +135,85 @@ _:bart-copies-assignment-2
   a acl:AccessControl ;
   acp:apply </myAccessPolicies#bart-copies-my-homework> .
 ```
+
+## Minimal Extension to WAC enabling the same features
+
+Another way of getting the same effect as ACP is to extend WAC with a `:authorizes`
+relation defined as
+
+```Turtle
+acl:accessTo owl:inverseOf [
+        owl:propertyChainAxiom ( acl:accessControl :authorizes )
+      ] .
+```
+
+(We leave the namespace for `:authorizes` open for the moment as it could be in the wac, acp, or another namespace).
+
+This would allow one to place rules in different resources without needing to specify the `wac:accessTo` relation as in ACP. So for example, the access control resource associated with Tim Berners-Lee's WebID Profile could contain
+
+```Turtle
+<> :authorizes _:a1, <personal#Rule1> .
+_:a1 a acl:Authorization;
+   acl:agentClass foaf:Agent ;
+   acl:mode acl:Read .
+```
+
+Note that the `acl:accessTo` relation is missing here. But it can easily be inferred. Any agent - be it a client or the server Guard - following the `Link: <card.acl>;rel="http://www.w3.org/ns/auth/acl#accessControl"` header from the original resource `<card>`, can then follow the `:authorizes` links and so deduce the following 2 statements (see the illustration below)
```

Comment:
```suggestion
Note that the `acl:accessTo` relation is missing here, but it can easily be inferred. Any agent &mdash; be it a client or the server Guard &mdash; following the `Link: <card.acl>;rel="http://www.w3.org/ns/auth/acl#accessControl"` header from the original resource `<card>`, can then follow the `:authorizes` links and so deduce the following 2 statements (see the illustration below):
```